src/worker.js

```diff
@@ -117,6 +122,26 @@ self.Module = {
             self.Module.cwrap('GDALGetRasterYSize', 'number', ['number']),
             errorHandling
         );
+        registry.GDALGetRasterMinimum = wGDALGetRasterMinimum(
+            self.Module.cwrap('GDALGetRasterMinimum', 'number', ['number']),
+            errorHandling
+        );
+        registry.GDALGetRasterMaximum = wGDALGetRasterMaximum(
+            self.Module.cwrap('GDALGetRasterMaximum', 'number', ['number']),
+            errorHandling
+        );
+        registry.GDALGetRasterNoDataValue = wGDALGetRasterNoDataValue(
+            self.Module.cwrap('GDALGetRasterNoDataValue', 'number', ['number']),
```

Comment:
Just double checking that this return type is always a number and not null, correct? Or does 'number' here include null?

Reply:
Yep, that's correct, it's always a number and never null (the return type from the [C API is `double`](https://gdal.org/api/raster_c_api.html#_CPPv424GDALGetRasterNoDataValue15GDALRasterBandHPi). Though, even if there was a possibility for null here, the translation between C types and Javascript types means that the return type would still end up being `number` because C expresses null pointers with what looks to Javascript like the number `0`. In that case we'd have to do some extra work on this end to check for `0` and then convert that to `null` as appropriate.